# This file is distributed under the same license as the Django package.
#
msgid ""
msgstr ""
"Project-Id-Version: Django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2018-01-12 12:35+0100\n"
"PO-Revision-Date: 2010-05-13 15:35+0200\n"
"Last-Translator: Django team\n"
"Language-Team: English <en@li.org>\n"
"Language: en\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: env/lib/python3.5/site-packages/django/contrib/gis/apps.py:8
msgid "GIS"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:64
msgid "The base GIS field."
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:199
msgid ""
"The base Geometry field -- maps to the OpenGIS Specification Geometry type."
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:277
msgid "Point"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:284
msgid "Line string"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:291
msgid "Polygon"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:298
msgid "Multi-point"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:305
msgid "Multi-line string"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:312
msgid "Multi polygon"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:319
msgid "Geometry collection"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:325
msgid "Extent Aggregate Field"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/db/models/fields.py:340
msgid "Raster Field"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/forms/fields.py:18
msgid "No geometry value provided."
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/forms/fields.py:19
msgid "Invalid geometry value."
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/forms/fields.py:20
msgid "Invalid geometry type."
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/forms/fields.py:21
msgid ""
"An error occurred when transforming the geometry to the SRID of the geometry "
"form field."
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/templates/gis/admin/openlayers.html:35
#: env/lib/python3.5/site-packages/django/contrib/gis/templates/gis/openlayers.html:12
msgid "Delete all Features"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/templates/gis/admin/openlayers.html:37
msgid "WKT debugging window:"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/templates/gis/openlayers.html:13
msgid "Debugging window (serialized value)"
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/views.py:8
msgid "No feeds are registered."
msgstr ""

#: env/lib/python3.5/site-packages/django/contrib/gis/views.py:14
#, python-format
msgid "Slug %r isn't registered."
msgstr ""
